fastlane_core/lib/fastlane_core/update_checker/update_checker.rb

```diff
@@ -186,14 +198,7 @@ def self.p_hash(args, gem_name)
         app_identifier = ios_app_identifier(args)
         @platform = :ios if app_identifier
       end
-
-      if app_identifier
-        return Digest::SHA256.hexdigest("p#{app_identifier}fastlan3_SAlt") # hashed + salted the bundle identifier
-      end
-
-      return nil
-    rescue
-      return nil
+      app_identifier
```

Comment:
I would add an explicit return here, just to make it clear you intend the output to be used